tests/network.rs

```diff
@@ -80,15 +89,25 @@ async fn test_broadcast() {
 // - Node2: set Node1 as bootnode, listens to subscribed topics
 #[tokio::test]
 async fn test_send_to() {
-    let (address_1, node_1, _) = node(30003, vec![], None, vec![]).await;
+    let keypair_1 = ed25519::Keypair::generate();
+    let address_1 = keypair_1.public().to_bytes();
+
+    let keypair_2 = ed25519::Keypair::generate();
+    let address_2 = keypair_2.public().to_bytes();
+
+    let (node_1, _message_receiver_1) = node(
+        keypair_1, 
+        30003, 
+        vec![], 
+        vec![]
+    ).await;
 
-    let (address_2, _node_2, receiver_gate) = node(
+    let (_node_2, message_receiver_2) = node(
+        keypair_2,
         30004,
-        vec![Peer::new(address_1, Ipv4Addr::new(127, 0, 0, 1), 30003)],
-        None,
-        vec![],
-    )
-    .await;
+        vec![(address_1, Ipv4Addr::new(127, 0, 0, 1), 30003)],
+        vec![Topic::HotStuffRsSend(address_2)]
```

Comment:
Don't the nodes subscribe to their own mailbox topic from the start? Shouldnt need to do it explicitly here?

Reply:
thanks for pointing that out, you're right, its already been subscribed on instantiation of new behaviour, will remove explicit declaration of mailbox topics